Packs/CommonScripts/Scripts/StixCreator/StixCreator.py

```diff
@@ -83,6 +85,45 @@
 }
 
 
+def search_relationship_indicators(value: str) -> list[dict]:
+    relationships = demisto.searchRelationships({"entities": [value]}).get("data", [])
+
+    query = ""
+    for rel in relationships:
+        entity_a = rel.get("entityA", "")
+        entity_b = rel.get("entityB", "")
+        if entity_a.lower() == value.lower():
+            query += f'"{entity_b}"'
+        else:
+            query += f'"{entity_a}"'
```

Comment:
I wold check also here if entity_b == value.lower before I use it, in case we are getting an unrelated relationship and if this is the case, do not handle it and log that we got an irrelevant relation ship with all the information(its indicator that was queried and the relation ship returned)